### ISSUE TITLE
bug(MatMenu): Material menu on expansion not announcing "expanded" when using a screen reader

### ISSUE THREAD

**Issue by yashwantyashu** [angular/components#29723]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

The angular material menu button on focus is announced as menu button collapsed by NVDA or similar screen reader software and in speech viewer it is logged. However, on trigger opening the button using enter key, the expanded state is not announced by the screen reader software nor logged in the speech viewer.
![Screenshot (155)](https://github.com/user-attachments/assets/2d4144ac-6630-4819-bab6-3f976d480795)

### Reproduction

Link:https://material.angular.io/components/menu/overview
Steps to reproduce:
1. Open the link.
2. Install NVDA or other similar screen reader software and open speech viewer to see the logs.
3. Tab through the interactive elements until it lands on the menu button inside basic menu container
4. Observe the screen reader announcement and speech viewer log.(announces collapsed)
5. Press enter key to open the menu.
6. Observe the screen reader announcement and speech viewer log.(does not announce expanded)

### Expected Behavior

When the user open the menu button using enter key, the screen reader should notify the user about the state change to expanded followed by the submenu items. Can find such similar behavior in mat-select, mat-expansion panels but not in mat-menu.

### Actual Behavior

When the user open the menu button using enter key, the screen reader does not notify the user about the state change to expanded and only announces the submenu items. 

### Environment

- Angular:
- CDK/Material:
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu):

**Comment by crisbeto** [comment]
I suspect that it doesn't get announced, because focus moves from the trigger into the menu so the screen reader stops monitoring the trigger.

**Comment by yashwantyashu** [comment]
Hi, 
Is this issue fixed?

**Comment by crisbeto** [comment]
We're setting `aria-expanded` correctly so I don't think there's anything more to do on our end. It's up to the screen reader whether or not it gets announced.

**Comment by yashwantyashu** [comment]
Right. 
Is there any suggested work around in the code while using mat-menu to get it announced?

**Comment by crisbeto** [comment]
You can announce the expanded state yourself using the `LiveAnnouncer` service from the CDK.

**Comment by em843** [comment]
Seeing this issue with Mat Select as well
